antiphishing/antiphishing.py

```
@@ -1,357 +1,765 @@
+            try:
+                formatted_info = "\n".join(f"**{key.replace('_', ' ').title()}**: {value}" for key, value in additional_info.items())
+                if len(formatted_info) > 1000:
+                    formatted_info = formatted_info[:1000] + "\n... (truncated)"
+                log_embed.add_field(name="Additional Info (V2)", value=formatted_info, inline=False)
+            except Exception as e:
+                log.error(f"Error formatting V2 additional info for log: {e}")
+
+        log_embed.set_footer(text=f"User total detections: {await self.config.member(message.author).caught()}")
+
+        staff_mention = f"<@&{staff_role_id}>" if staff_role_id else ""
+        allowed_mentions = discord.AllowedMentions(roles=True if staff_role_id else False)
```

Ruff (0.8.2)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)